js/bloodmallet_standard.js

```diff
@@ -3624,6 +3680,24 @@ function search_iconized_chart_cookie() {
   document.getElementById("iconized_charts_checkbox").checked = whTooltips.iconizeLinks;
 }
 
+/**
+ * Sets the relative azerite cookie
+ */
+function set_relative_azerite_cookie(){
+  if(debug)
+    console.log("set_relative_azerite_cookie");
+  Cookies.set("bloodmallet_relative_azerite", relative_azerite_string, {expires: 31, path: ''})
```

Comment:
`;`

Reply:
"bloodmallet_relative_azerite" -> "bloodmallet_relative_azerite_forge_import_string"

I know, that at least I could get confused by it otherwise and would think this would affect the chart in some way at some point in the future.